src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall().toString());
+        System.out.println(lottery.getRandomBall().toString());
+        System.out.println(lottery.getRandomBall().toString());
```

Comment:
You can solve this task using only one for-loop. Imagine 100 balls, are you going to code 100 lines of `System.out.println(lottery.getRandomBall().toString());`?